modules/extensions-spring-support/src/test/resources/models/metadata.json

```diff
@@ -38647,7 +38647,6 @@
       "format": "xml",
       "type": "Object",
       "annotations": {
-        "typeId": "#root:shiporder",
```

Comment:
Why was this moved?

Reply:
I'm not sure, this happens when the extension model is regenerated. It is something that is already in the master. I will create a ticket to review it.
Also, I reverted the changes to this file.